### PR TITLE
Update the expense_rates table

### PR DESCRIPTION
**Pull request by Ben-Edwards-cgi**
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/main/resources/db/migration/V1_89__expense_updates.sql [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+   FROM juror_mod.appearance a
+   GROUP BY a.juror_number, a.pool_number, a.loc_code;
+
+-- juror_mod.juror_expense_totals source
+
+CREATE OR REPLACE VIEW juror_mod.juror_expense_totals
+AS SELECT jes.juror_number,
+          jes.pool_number,
+          j.first_name,
+          j.last_name,
```
Comment (CCHAPMAN01) [inline src/main/resources/db/migration/V1_89__expense_updates.sql:60]: have these intentionally been left here? assuming performance is Ok just dropping them from the previous query?

**Review by CCHAPMAN01 — COMMENTED** [review]
Just one comment to clarify please 😄

File: src/main/resources/db/migration/V1_89__expense_updates.sql (hunk 1 above)
Comment (Ben-Edwards-cgi) [reply src/main/resources/db/migration/V1_89__expense_updates.sql:60]: We Dropped them from the sub-totals query and added them to the totals query  (As this makes the query run a little faster :))

**Review by CCHAPMAN01 — APPROVED** [review]
Happy with the changes - PR showing conflicts which need resolving prior to merging 👍

**Pull request merged**